app/helpers/customers_helper.rb

```diff
@@ -0,0 +1,7 @@
+module CustomersHelper
+  def tiers_array_for_select_box
```

Comment:
A little helper to help provide the option label text of the tiers select box with much more information. Shows the Tier's name, price and quantity together when choosing your usage tier.

<img width="368" src="https://user-images.githubusercontent.com/1930474/41760326-07aabb1e-75b0-11e8-9433-ea699720fc29.png">